tajo-common/src/main/java/org/apache/tajo/type/Type.java

```diff
@@ -36,6 +36,7 @@
 
   // No paramter types
   public static final Any Any = new Any();
+  public static final Bit Bit = new Bit();
```

Comment:
IMO, the main difference between singleton with (typical) ``getInstance`` method and ```static final``` variable is the initialization time. Singleton pattern is preferred if the variable needs to be initialized lazily. Static variable is initialized at class loading. In this case, they are always used throughout a system. So, IMO, static variable is also right choice here. In addition, static variable provides more concise usages.